integrationtests/src/test/java/org/axonframework/integrationtests/testsuite/administration/events/PersonEvent.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2010-2025. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.integrationtests.testsuite.administration.events;
+
+import org.axonframework.eventsourcing.annotations.EventTag;
+import org.axonframework.integrationtests.testsuite.administration.common.PersonIdentifier;
+
+public interface PersonEvent {
+    PersonIdentifier identifier();
+
+    @EventTag(key = "Person")
+    default String tag() {
+        return identifier().identifier();
```

Comment:
I know it's just a test but it looks weird. Maybe we should change the `PersonIntendifier` identifier to key?